documentation/docs/examples/cad-nodefiltering.mdx

```diff
@@ -45,10 +45,29 @@ sdk.revisions3D.list3DNodes(model.modelId, model.revisionId,
     properties: { PDMS: { Function: 'EA' } }
   })
   .autoPagingEach(node => {
-    model.showNodeByTreeIndex(node.treeIndex, true);
+    model.showNodeByTreeIndex(node.treeIndex, applyToChildren = true);
   });
 ```
 
+For the example model, which is quite small, this performs well. For larger assets we might improve the performance 
+by avoiding use of `applyToChildren = true` in `Cognite3DModel.showNodeByTreeIndex` which leads to a network request.
+This can be done by using the `node.subtreeSize`-field in the response from `list3DNodes`:
```

Comment:
```suggestion
For the example model, which is quite small, this performs well. For larger assets, we might improve the performance 
by avoiding use of `applyToChildren = true` in `Cognite3DModel.showNodeByTreeIndex` which leads to a network request.
This can be done by using the `node.subtreeSize` – a field in the response from `list3DNodes`:
```